test-sites/ocw-ci-test-www/README.md

```diff
@@ -0,0 +1 @@
+# ocw-ci-test-www
```

Comment:
I have not written very many tests for `www` yet, and this is the only file in the test site.

Really I included `www` mostly as a proof-of-concept for building / running tests against multiple sites using multiple hugo configs.